### PR TITLE
[Runtime] Update runtime.rst

### PR DESCRIPTION
**Pull request by morrsky**
typo PHP-PM:PHP-FPM

### CODE REVIEW

**Comment by javiereguiluz** [comment]
Good catch @morrsky! Thanks and congrats on your first Symfony Docs contribution!

File: components/runtime.rst [hunk 1]
```diff
@@ -2,7 +2,7 @@ The Runtime Component
 =====================
 
     The Runtime Component decouples the bootstrapping logic from any global state
-    to make sure the application can run with runtimes like PHP-PM, ReactPHP,
+    to make sure the application can run with runtimes like PHP-FPM, ReactPHP,
```
Comment (xabbuh) [inline components/runtime.rst:5]: I think this change is wrong and PHP-PM refers to the [PHP Process Manager](https://github.com/php-pm/php-pm).

Comment (xabbuh) [reply components/runtime.rst:5]: @Nyholm Can you help here as you contributed the initial version of this document?

Comment (Nyholm) [reply components/runtime.rst:5]: Correct, It is meant to say PHP-PM.

Comment (xabbuh) [reply components/runtime.rst:5]: see #19221 where I try to clarify this by adding links to these projects

Comment (javiereguiluz) [reply components/runtime.rst:5]: Sorry about this! I just merged your PR fixing this.

**Pull request merged**